src/scroll-timeline-css-parser.js

```diff
@@ -204,12 +204,34 @@ export class StyleParser {
               p
             );
           }
+
+          // If there is no duration, animationstart will not happen,
+          // and polyfill will not work which is based on animationstart.
+          // Add 1s as duration to fix this.
+          if(timelineName || hasAnimationTimeline) {
+            if(!this.hasDuration(shorthand)) {
+              rule.block.contents = rule.block.contents.replace(
+                "animation:",
+                "animation: 1s "
```

Comment:
Please add a comment that we should track whether the duration is artificial so that we can later  track that we need to update timing to properly see duration as 'auto' for the polyfill.